content/src/fieldConfig/profile-info-oscar.json

```diff
@@ -47,6 +47,12 @@
         "header": "",
         "description": "",
         "errorTextRequired": ""
+      },
+      "nexusDbaName": {
+        "placeholder": "",
+        "header": "",
+        "errorTextRequired": "",
+        "description": ""
       }
```

Comment:
Even though it won't be used by Oscar, I believe you also need to add the nexusBusinessName block and properties here